src/NServiceBus.Ninject.AcceptanceTests/NServiceBus.Ninject.AcceptanceTests.csproj

```diff
@@ -7,7 +7,7 @@
     <ProjectGuid>{F4A36825-25A7-4855-9C1A-2FFBB68E187A}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>NServiceBus.Ninject.AcceptanceTests</RootNamespace>
```

Comment:
I shortened the default namespace because with that length it's basically impossible to have a test which doesn't exceed msmq max. queue length.